app/components/dashboard/DcChart.vue

```
@@ -437,13 +437,9 @@
           </div>
 
           <!-- DC_PANEL Year -->
-          <div id="panel_year" class="bg-theme-col2 dc_panel drag" :style="pnl.year.style" v-show="pnl.year.is_show">
+          <div id="panel_year" class="bg-theme-col2 dc_panel drag panel_year" :style="pnl.year.style" v-show="pnl.year.is_show">
```

ESLint

[error] 'v-show' should be on a new line.

(vue/max-attributes-per-line)